ChimpAndSee/js/main.js

```diff
@@ -1 +1,9 @@
+const checkPath = function() {
```

Comment:
```suggestion
function checkPath() {
```
Minor thing, but declaring `checkPath` as a function, not a variable, allows you to take advantage of [hoisting](https://developer.mozilla.org/en-US/docs/Glossary/Hoisting).